Libraries/LibWeb/WebGL/WebGLFramebuffer.cpp

```diff
@@ -4,15 +4,21 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <LibJS/Runtime/Realm.h>
 #include <LibWeb/Bindings/WebGLFramebufferPrototype.h>
 #include <LibWeb/WebGL/WebGLFramebuffer.h>
 
 namespace Web::WebGL {
 
 GC_DEFINE_ALLOCATOR(WebGLFramebuffer);
 
-WebGLFramebuffer::WebGLFramebuffer(JS::Realm& realm)
-    : WebGLObject(realm, 0)
+GC::Ptr<WebGLFramebuffer> WebGLFramebuffer::create(JS::Realm& realm, GLuint handle)
+{
+    return realm.heap().allocate<WebGLFramebuffer>(realm, handle);
```

Comment:
Do we not need any realm based initialization to set prototypes etc 🤔 Doesn't look like the subclasses have them currently, but they are platform objects defined in spec? If they do - then this should be `realm.create<WebGLFramebuffer>` instead

Reply:
Oh, I see this was changed in a later commit!

Reply:
Made them return GC::Ref :^)